csb/src/main/scala/edu/msstate/dasi/csb/OptionParser.scala

```diff
@@ -103,9 +92,9 @@ class OptionParser(override val programName: String, programVersion: String, con
 
       opt[String]('l', "log")
```

Comment:
We actually don't need the `log` option anymore in the `synth` scope since the `DataDistributions` is now made from a graph.